granary/test/test_facebook.py

```diff
@@ -831,7 +831,7 @@ def tag_uri(name):
 </a>
 <span class="summary">my link caption</span>
 </p>
-<span class="p-location h-card h-as-location">
+<span class="p-location h-card h-geo">
```

Comment:
I don't think this is exactly semantically correct per [h-card](http://microformats.org/wiki/h-card). 

It says h-card commonly embeds a "p-geo h-geo" or "p-adr h-adr", but I don't think it is meaningful for one thing to be both an "h-card" and an "h-geo".